### ISSUE TITLE
Tape tests fail to run under Windows

### ISSUE THREAD

**Issue by SaintPeter** [FreeCodeCamp/COM1000#14]
I took a deeper look into this error message:

```
$ npm test

> COM1000@0.0.0 test d:\mean\COM1000
> node node_modules/.bin/babel-tape-runner tests/**/*.js | faucet

d:\mean\COM1000\node_modules\.bin\babel-tape-runner:4
case `uname` in
^^^^

SyntaxError: Unexpected token case
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:414:25)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:467:10)
    at startup (node.js:136:18)
    at node.js:963:3
 not ok 1 no plan found
not ok 2 no assertions found
⨯ fail  2
npm ERR! Test failed.  See above for more details.
```

And realized that the command is trying to run a shell script with node. . . :smile: 

The correct command, I think, is:
`node_modules/.bin/babel-tape-runner tests/**/*.js | node node_modules/faucet`, but I don't get any output.

Running without faucet gives me:

```
$ node_modules/.bin/babel-tape-runner tests/**/*.js
TAP version 13
# #reducer.spec
# Test 1 should return a default state if none is provided
ok 1 should be equivalent
# Test 2 should return a new state upon a valid action type
ok 2 should not be equal
# Test 3 should return previous state on invalid action
ok 3 should be equivalent
# #tape.spec
# Test 1 should fail
# Assertion passes
ok 4 should be equal
# Assertion fails
not ok 5 should not be equal
  ---
    operator: notEqual
    expected: undefined
    actual:   5
  ...

1..5
# tests 5
# pass  4
# fail  1
```

**Comment by terakilobyte** [comment]
I can't reproduce this locally. I do not get the error you do on Windows as I am on a Mac.

See comment below.

**Comment by terakilobyte** [comment]
See if tap-spec works on windows. `npm install --save-dev tap-spec` and replace the pipe to `| faucet` with `| tap-spec` in the test script in package.json. Also, can windows even use the pipe operator?

**Comment by SaintPeter** [comment]
I was able to get faucet working by installing it globally:

```
$ node_modules/.bin/babel-tape-runner tests/**/*.js | faucet
✓ #reducer.spec
✓ Test 1 should return a default state if none is provided
✓ Test 2 should return a new state upon a valid action type
✓ Test 3 should return previous state on invalid action
✓ #tape.spec
✓ Test 1 should fail
✓ Assertion passes
⨯ Assertion fails
  not ok 5 should not be equal
    ---
      operator: notEqual
      expected: undefined
      actual:   5
    ...

# tests 5
# pass  4
⨯ fail  1
```

I still can't run a shell script with node.  My command line:
`node_modules/.bin/babel-tape-runner tests/**/*.js | faucet`

And, yes, Windows is a real grown up operating system can can use the pipe operator.

**Comment by SaintPeter** [comment]
Fixed by #17
